code/modules/hydroponics/eggincubator.dm

```diff
@@ -48,10 +49,21 @@
 /obj/machinery/egg_incubator/attackby(var/obj/item/O as obj, var/mob/user as mob)
 	if(..())
 		return 1
-	if(contents.len >= limit)
-		to_chat(user, "\The [src] has no more space!")
-		return 1
-	if (istype(O,input_path))
+	if(ismultitool(O) && !panel_open)
```

Comment:
The fact that this is now copypasted over so many machines is bad for maintainability. Convert it into a machine flag, please.

Reply:
Like, on every machine?

Reply:
Yeah, move it up to machinery, you can use SCREWTOGGLE and WRENCHANCHOR as good examples

Reply:
No I mean are you asking me to replace all the instances of this copy paste within this PR? Or just make the flag for this machine specifically?

Reply:
Please do delete earlier pasting in other machines that use it and replace with giving them the flag

Reply:
I feel as if that's a little bit much for my silly egg PR but fine fine

Reply:
I'm asking you to do nothing more than take the code you've already pasted here, and move it into machinery, then delete some code blocks and add a flag to other files. In terms of labor that's like three presses of delete, one cut/paste, and about thirty characters of writing based on what you pick as your flag name.

Reply:
There's not actually any other machines besides this and R&D machines that have need for this, unless I'm misunderstanding what you mean.